api_tests/revisions/views/test_schema_response_list.py

```diff
@@ -0,0 +1,84 @@
+
+import pytest
+
+from osf_tests.factories import (
+    SchemaResponsesFactory,
+    RegistrationSchemaFactory,
+    AuthUserFactory,
+    RegistrationFactory
+)
+
+from django.contrib.contenttypes.models import ContentType
+
+from osf.models import SchemaResponses
+
+
+@pytest.mark.django_db
+class TestSchemaResponseList:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def payload(self, node):
+        return {
+            'data': {
+                'type': 'schema_responses',
+                'attributes': {
+                    'title': 'new title'
+                },
+                'relationships': {
+                    'node': {
```

Comment:
Contract is expecting "registration" here. Is this just demoing that we can use arbitrary types?